tests/experimental/datasources/test_config.py

```diff
@@ -76,17 +76,51 @@
     "datasources": {
         "my_ds": {
             "name": "my_ds",
-            "type": "postgres",
-            "connection_string": "postgres",
+            "type": "sql",
+            "connection_string": "sqlite://",
         }
     }
 }
 
+WITH_OLD_STYLE_DS_DICT = {
```

Comment:
nit: prefer `OLD_STYLE_DS_DICT`. Seems more consistent with other dict names, eg `SIMPLE_DS_DICT`.

Reply:
I called it `WITH` because it has a new style config with the old style, it's not just the old style config.
However, if you still think this is clearer I don't mind changing it.

Reply:
`COMBINED_ZEP_AND_OLD_STYLE_DICT`